src/emulator/xlHeap.c

```diff
@@ -28,6 +29,7 @@ u32* gnHeapOS[HEAP_COUNT];
 //! TODO: these need better names
 #define CHKSUM_HI(v) ((u32)((v) >> 26))
 #define CHKSUM_LO(v) ((u32)((v) & 0x3F))
+#define HEAP_ALIGN(v, a) (((u32)(v) + a - 1) & ~(a - 1))
```

Comment:
sorry, missed this. Does ROUND_UP work instead here too?

Reply:
yeah it works, I just had to add a cast where it's used